.github/workflows/lint-test.yml

```
@@ -151,6 +151,16 @@ jobs:
         pip install -U build
         python -m build
 
+    # run the libcst parsers and check for errors
+    - name: libcst codemod
```

```suggestion
    - name: libcst codemod
      env:
        LIBCST_PARSER_TYPE: "native"
```
seems quite a bit faster, 0m47s (native) vs 1m15s (builtin) on my machine (11s vs 15s real time, with 8 cores)

---

https://github.com/DisnakeDev/disnake/pull/708/commits/81dd94e09ab2c49fc8d7dad7321f69718f8971ac